
DepthSFL_resnet18_on_cifar10_with_50_users_500_epochs_seed_123_split_[1, 2, 3].txt
Traceback (most recent call last):
  File "/home/gyoon814/research/HeteroSFL/main.py", line 93, in <module>
    ax_model_select = auxiliary_models[model_idx]
NameError: name 'auxiliary_models' is not defined